edgexfoundry/edgex-go
# build: Update to go-mod-messaging with single broker host info

Internal changes only, does not impact MessageQue configuration

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
Run non-secure edgex stack and stop core data
Build core data with this branch and branches from https://github.com/edgexfoundry/go-mod-messaging/pull/200 and https://github.com/edgexfoundry/go-mod-bootstrap/pull/407
Run local core data with DEBUG logging
Verify Core data logs show events received from MessageBus

## New Dependency Instructions (If applicable)
